tests/simulations.ts

```diff
@@ -205,6 +205,11 @@
                 qualifier: "contains",
                 test: "folds from line XXXX to line 2"
             },
+            {
+                command: `diff source:"${projectPath}tests${sep}diffbase${sep}diff_html_diffSpaceIgnore.txt" diff:"${projectPath}tests${sep}diffnew${sep}diff_html_diffSpaceIgnore.txt" read_method:file`,
+                qualifier: "contains",
+                test: `${text.green}<div${text.diffchar}class="h${text.clear}i${text.diffchar}yabubbaye"id="hello"${text.clear}>${text.diffchar}Hi,world!${text.clear}`

```

Comment:
Instead of text.clear use **text.none**.  I bet that will solve the problem.  The text.clear is `\u001b[24m\u001b[22m` which means remove underline and use normal color.  I don't think I am using that anywhere and should remove it.

https://en.wikipedia.org/wiki/ANSI_escape_code

Reply:
Wow... I was about to paste a comment here and saw my problem. I was using [node debugging (--inspect-brk)](https://nodejs.org/en/docs/guides/debugging-getting-started/#command-line-options) with a breakpoint in `services.ts` to capture the failing `stdout` message and missed that escape sequences \u001b were hiding in there! Hurray!! I'll push with a properly failing test here in a few.